Title Screen Fix

Zyra915 opened the pull request:
Fixed Title screen and implemented freezing when paused. Also fixed merge issues

Zyra915 committed:
Title Screen Fix

Fixed Title screen and implemented freezing when paused

zachj112 reviewed (changes requested):
Going from start -> title screen -> start makes the start button unable to be pressed fyi

Zyra915 committed:
Fixed Button Issues, Added Menus

Fixed Button Issues with previous pull request. Moved menus from the testing scene to the MovementTestScene
